web/src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<web-app version="3.1"
+         xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd"
+         metadata-complete="false">
+
+  <env-entry>
+    <description>The client id assigned to the developer github application</description>
+    <env-entry-name>java:global/GITHUB_CLIENT_ID</env-entry-name>
+    <env-entry-type>java.lang.String</env-entry-type>
+    <env-entry-value>c3c3ec74379823dea3d7</env-entry-value>
+  </env-entry>
+  <env-entry>
+    <description>The client secret assigned to the developer github application</description>
+    <env-entry-name>java:global/CLIENT_SECRET</env-entry-name>
+    <env-entry-type>java.lang.String</env-entry-type>
+    <env-entry-value>284d925b55ab7e3931ea6b4325e0e6fbbf3c532b</env-entry-value>
```

Comment:
Right, this is for a test dev app in my account right now. I need to come
up with a way to take this info from build env when the war is built. Have
to look up property replacement in war plugin.
On Apr 6, 2016 01:38, "Andrew Lee Rubinger" <notifications@github.com>
wrote:

> In web/src/main/webapp/WEB-INF/web.xml
> <https://github.com/redhat-developer-tooling/katapult/pull/19#discussion_r58669958>
> :
>
> > +         xmlns="http://xmlns.jcp.org/xml/ns/javaee"
> > +         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
> > +         xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd"
> > +         metadata-complete="false">
> > +
> > +  <env-entry>
> > +    <description>The client id assigned to the developer github application</description>
> > +    <env-entry-name>java:global/GITHUB_CLIENT_ID</env-entry-name>
> > +    <env-entry-type>java.lang.String</env-entry-type>
> > +    <env-entry-value>c3c3ec74379823dea3d7</env-entry-value>
> > +  </env-entry>
> > +  <env-entry>
> > +    <description>The client secret assigned to the developer github application</description>
> > +    <env-entry-name>java:global/CLIENT_SECRET</env-entry-name>
> > +    <env-entry-type>java.lang.String</env-entry-type>
> > +    <env-entry-value>284d925b55ab7e3931ea6b4325e0e6fbbf3c532b</env-entry-value>
>
> I think this can't be made public and stored in the repo. :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/redhat-developer-tooling/katapult/pull/19/files/9500427278f226e637b151d535711dc85c796dca#r58669958>
>